pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
I was thinking about this since yesterday.  I think we should add a `PythonDefaults` class in `com/linkedin/gradle/python/extension`.   We'll put the Python 2 and 3 defaults, as well as the whitelist and the logic for normalization there.  `PythonDetails` will create and manage these and handle passing them to the `PythonVersion` class as necessary.  I think that'll be a nice separation of concerns and should facilitate good testing and the described use cases.

Any objections if I write `PythonDefaults` in Groovy?